tests_integration/test_temporary_connection.py

```diff
@@ -19,39 +21,69 @@
 
 @pytest.mark.integration
 @pytest.mark.no_qa
-@mock.patch.dict(
-    os.environ,
-    {
-        "SNOWFLAKE_CONNECTIONS_INTEGRATION_ACCOUNT": os.environ.get(
-            "SNOWFLAKE_CONNECTIONS_INTEGRATION_ACCOUNT", None
-        ),
-        "SNOWFLAKE_CONNECTIONS_INTEGRATION_USER": os.environ.get(
-            "SNOWFLAKE_CONNECTIONS_INTEGRATION_USER", None
-        ),
-        "SNOWFLAKE_CONNECTIONS_INTEGRATION_PRIVATE_KEY_FILE": os.environ.get(
-            "SNOWFLAKE_CONNECTIONS_INTEGRATION_PRIVATE_KEY_PATH",
-            os.environ.get("SNOWFLAKE_CONNECTIONS_INTEGRATION_PRIVATE_KEY_FILE"),
-        ),
-    },
-    clear=True,
-)
 def test_temporary_connection(runner, snapshot):
+    with mock.patch.dict(os.environ, _prepare_env_variables(), clear=True):
```

Comment:
It could be a fixture. 
Even used one time this could make test more readable.

Reply:
Changed to fixture.